components/ActivityIndicator/examples/ActivityIndicatorExample.swift

```diff
@@ -1,36 +1,42 @@
 /*
  Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+ 
```

Comment:
Sorry, I didn't realize this was an existing file. Please remove the newlines and restore the copyright to the way it was originally.  @ianegordon may know better, but I believe it's best not to touch these once they're committed.